src/test/java/seedu/address/model/contact/OrganizationTest.java

```diff
@@ -25,29 +26,24 @@ public void asObservableList_modifyList_throwsUnsupportedOperationException() {
     }
 
     @Test
-    public void isSameContact() {
+    public void isSameOrganization() {
```

Comment:
I would say since test methods usually refer the name of the method being tested, this probably should be kept as `isSameContact`.

Reply:
Also perhaps this method should be testing organisations specifically since it'd be entirely equivalent to the one defined in `Contact` otherwise.

Reply:
Hm yeah I’ll have to modify the typical contacts to include Organization and Recruiter examples